src/infinite-list/index.tsx

```diff
@@ -101,15 +117,18 @@ const InfiniteList = (props: InfiniteListProps, ref: any) => {
 
   const onMomentumScrollEnd = useCallback(
     event => {
-      if (isOnEdge.current) {
-        onReachEdge?.(pageIndex.current!);
-        reloadPagesDebounce?.(pageIndex.current);
-      } else if (isNearEdge.current) {
-        reloadPagesDebounce?.(pageIndex.current);
-        onReachNearEdge?.(pageIndex.current!);
+      console.warn('onMomentumScrollEnd: ', pageIndex.current);
+      if (pageIndex.current) {
```

Comment:
What's the purpose of checking `if (pageIndex.current)` here?

Reply:
Not to trigger callbacks with undefined param

Reply:
But should pageIndex be undefined at all? 
I see we update its value in `onScroll`, how do we get to `onMomentumScrollEnd` without a scroll happening? 
Moreover, should we maybe initialize the `pageIndex` ref with a value to avoid an undefined value?

Reply:
There is an issue with `onMomentumScrollEnd` on Android (not being called) and keeping the `pageIndex` undefined in the init was to avoid some code in the `onScroll` but still execute the rest. I don't want to change it unless we recheck the whole scenario, but we can do it later.